examples/some-issuer/templates/discord-oauth.hbs

```diff
@@ -10,7 +10,8 @@
         username: '{{username}}',
         state,
       };
-      window.opener.postMessage(message, '{{dapp_domain}}/');
+      window.opener.postMessage(message, '{{dapp_domain}}');
+      window.opener.postMessage(message, '{{verifier_dapp_domain}}');
```

Comment:
Is it correct that only one of these will actually send the message, depending on whether the popup originated from the issuer or verifier frontend?

Reply:
Both are tried every time. It results in an error in the console, as one of them don't match. Can see if I can find another way around, but it works fine it its current state.